articles/index.md

```diff
@@ -68,37 +68,37 @@ Develop a [theory of change](/theory) for your activist organisation using our t
 
 <action-smart-large/>
 
-## We work with data & revolutionary tech
+## We are working with data & revolutionary technologies
 
 We don’t just write guides. Activist Handbook uses a **data-driven approach** and develops **digital tools** to empower change-makers.
 
 ### Large numbers, big impact
 
 - **🔍 1K+ search queries analysed:** We know what challenges activists face most, and what words they type into Google to find help. With this SEO data, we can write the guides that activists actually need and make sure our platform is found by the people looking for it.
 - **📚 4.7K resources:** We did not write our 450+ guides from scratch. Instead, we brought together material from experienced campaigned around the globe and organised all of it in into one single comprehensive handbook.
-- **🔥 4K+ activist trained every month:** Our number of volunteers can be counted on one hand and budget is €0. Yet, we manage to train thousands of activists every single month. And the more activists use and contribute to our wiki platform, the better it gets.
+- **🔥 4K+ activist trained every month:** Our number of volunteers can be counted on one hand and budget is €0. Yet, we manage to train thousands of activists every single month. Every person counts, and the more activists use and contribute to our wiki platform, the better it gets. Share your knowledge with others [right now](https://edit.activisthandbook.org/new)!
```

Comment:
Removing the ". And" part.

I am not sure about the second part. Very good line, it encourages people to contribute, and maybe we can add a concrete action to it, or point to an editing guide? It is possible they won't know how to contribute right away if they click the link, and will just return back to the landing page, but at least they will have a knowledge of how to do it, and how easy it is.

Reply:
Moved some content around, so this part doesn't actually exist anymore. But you can find back these words on our Introduction, About and Contribute pages now.